LICENSE

```diff
@@ -1,21 +1,108 @@
-MIT License
-
-Copyright (c) 2019, MIT Data To AI Lab
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+Business Source License 1.1
+
+Parameters
+
+Licensor:             DataCebo, Inc.
+
+Licensed Work:        SDGym
+                      The Licensed Work is (c) DataCebo, Inc.
+
+Additional Use Grant: You may make use of the Licensed Work, and derivatives of the Licensed
+                      Work, provided that you do not use the Licensed Work, or derivatives of
+                      the Licensed Work, for a Synthetic Data Benchmarking Service.
+
+                      A "Synthetic Data Benchmarking Service" is a commercial offering
+                     	that allows third parties (other than your employees and
+                     	contractors) to access the functionality of the Licensed
+                      Work so that such third parties directly benefit from the
+                      data specification, data transformation, machine learning, synthetic
+                      data creation, synthetic data evaluation or performance evaluation
+                      features of the Licensed Work.
+
+
+Change Date:          Change date is four years from release date.
+Please see https://github.com/sdv-dev/SDGym/releases
```

Comment:
This should spaced so it starts with the "Change date is four years from release date." sentence above

Reply:
I'm still seeing an issue with this @katxiao -- am I looking at the wrong version? The intended formatting is to add spaces before "Please see" to keep it inline. See image below.

<img width="570" alt="image" src="https://user-images.githubusercontent.com/3227804/209198437-124bd4c2-1d5b-4fd0-9c90-53ba4b146a8b.png">